Support heartbeatStrategy in autologin config

Nambers opened the pull request:
close #1867

Nambers committed:
support heartbeatStrategy in autologin config

Stardust-minus commented:
https://github.com/mamoe/mirai/blob/7811dc8f64ab0316ec9a67784bd0a918ae890585/mirai-console/backend/mirai-console/src/internal/data/builtins/AutoLoginConfig.kt#L42
考虑写上目前Mirai所支持所有的心跳方式以供用户选择。

Nambers committed:
doc: update tips

https://github.com/mamoe/mirai/blob/8d641c1d4c82c6792f29113cfd3978ba90a71af3/docs/Bots.md
